build.gradle

```diff
@@ -86,11 +92,33 @@ subprojects {
       '-ArequirePrefixInWarningSuppressions',
       '-AshowSuppressWarningKeys',
       "-Astubs=$rootDir/config/checker/",
-      '-AsuppressWarnings=all:type.anno.before.modifier',
+      '-AsuppressWarnings=all:type.anno.before.decl.anno,all:type.anno.before.modifier',
     ]
   }
 
 
+  // Lombok support
+  // Here comes the tricky part: there's a tradeoff...
```

Comment:
:see_no_evil:

Reply:
Let's see how https://github.com/rzwitserloot/lombok/issues/2433 goes, this might be the solution for us (specifically, remove the `Minus` for the `LOMBOK_PROVIDED_BY=regular-dependency` mode).